### PR TITLE
Upgrade the support library to 24.2.1

### PR DESCRIPTION
**Pull request by passsy**
Google recommends the upgrade:

> We do, however, recommend using 24.2.1 over 24.2.0 as it contains a number of bug fixes.
https://www.reddit.com/r/androiddev/comments/52m6ac/android_support_library_revision_2421_released/d7lptza

https://github.com/grandcentrix/ThirtyInch/issues/8

### CODE REVIEW

**Comment by StefMa** [comment]
LGTM 👍

File: plugin/build.gradle [hunk 1]
```diff
@@ -32,7 +32,7 @@ android {
 
 dependencies {
     compile project(':thirtyinch')
-    compile 'com.pascalwelsch.compositeandroid:activity:0.2.2'
+    compile "com.pascalwelsch.compositeandroid:activity:$supportLibraryVersion"
```
Comment (jannisveerkamp) [inline plugin/build.gradle]: Is this correct?

Comment (StefMa) [inline plugin/build.gradle]: According to this [documentation](https://github.com/passsy/CompositeAndroid#composite-android-solution), yes :)

Why have it worked before? 💭  :)

Comment (jannisveerkamp) [inline plugin/build.gradle]: Just was confused by this line 👍

Comment (passsy) [inline plugin/build.gradle]: compositeandroid introduced the supportlibrary versioning within the last week ;)

**Pull request merged**